src/components/Header/Header.jsx

```diff
@@ -1,25 +1,31 @@
+import React from "react";
 import { useNavigate } from "react-router-dom";
-
+import { useSelector, useDispatch } from "react-redux";
+import { removeUser } from "../../store/user/actions";
 import Button from "../../common/Button/Button";
 import Logo from "./components/Logo/Logo";
-
 import styles from "./Header.module.css";
-
-function Header({ userName, setUserName }) {
+import { BUTTON_TEXT, USER_LOGGED } from "./headerStrings";
+import { PATH_URIS, TOKEN } from "../../constants";
+function Header() {
   const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const userName = useSelector((state) => state.user.name);
 
   function onLogoutHandler() {
-    localStorage.removeItem("token");
-    setUserName("");
-    navigate("/login");
+    localStorage.removeItem(TOKEN);
```

Comment:
Move LS related functionality to `removeUser` action

Reply:
done